Scripts/Guests.js

```diff
@@ -0,0 +1,156 @@
+      + ' Sufism and she devotes her life to the spirituality of the world.',
+    img: 'Assets/AbidaP.png',
+  },
+  {
+    name: 'Deva Premal',
+    sub: 'Spritual and Sanskritist',
+    id: '1',
+    description:
+      'Deva Premal is a singer known for introducing Sanskrit mantras into the mainstream'
+      + 'Her meditative, spiritual music is produced with Prabhu Music',
+    img: 'Assets/Dava.jfif',
+  },
+  {
+    name: 'Roger Waters',
+    sub: 'CO Founder of pink floyd band',
+    id: '2',
+    description:
+      'George Roger Waters is an English musician, singer songwriter and composer'
+      + ' who is known for his work in the genre of Spritual music.',
+    img: 'Assets/RogerWaters.png',
+  },
+  {
+    name: 'Ludovico Einaudi',
+    sub: 'Italian Piano Composer',
+    id: '3',
+    description:
+      'Ludovico Maria Enrico Einaudi OMRI is an Italian pianist and composer.'
+      + 'Einaudi began his career as a classical composer, and world music',
+    img: 'Assets/pianoman.png',
+  },
+  {
+    name: 'Batzorig Vaanchig',
+    sub: 'Mongolian Throat Singner',
+    id: '4',
+    description:
+      'Batzorig Vaanchig Is an Mongolion Thorat Singer who specializes '
+      + ' the fire of the great Mongol  Leader Gengis Khan.',
+    img: 'Assets/Batzorig.jfif',
+  },
+  {
+    name: 'Andre Watts',
+    sub: 'American Singer',
+    id: '5',
+    description:
+      'Andre Watts is an American singer, songwriter, and musician. He is known for his'
+      + ' work in the genre of Classical Music.',
+    img: 'Assets/Andre.jfif',
+  },
+];
+
+const template = `
+<h3 class="type">Featured Artist</h3>
+<div class="line line1"></div>
+<section class="container">
+    <article class ="art 1">
+        <img src="${artist[0].img}" alt="${artist[0].name}" class="artist-img"> 
+        <div class ="arti">
+            <h4 class="artist-name">${artist[0].name}</h4>
+            <p class="sub">${artist[0].sub}</p>
+            <div class ="line line2"></div>
+            <p class="artist-description">${artist[0].description}</p>
+        </div>
+    </article>
+    <article class ="art 2">
+        <img src="${artist[1].img}" alt="${artist[1].name}" class="artist-img">
+        <div class ="arti">
+            <h4 class="artist-name">${artist[1].name}</h4>
+            <p class ="sub">${artist[1].sub}</p>
+            <div class ="line line2"></div>
+            <p class="artist-description">${artist[1].description}</p>
+        </div>
+    </article>
+    <button class="btn-more btn-sec" id="expand">MORE</button>
+    <article class ="art 3 more">
+        <img src="${artist[2].img}" alt="${artist[2].name}" class="artist-img">
+        <div class ="arti">
+            <h4 class="artist-name">${artist[2].name}</h4>
+            <p class ="sub"> ${artist[2].sub}</p>
+            <div class="line line2"></div>
+            <p class="artist-description">${artist[2].description}</p>
+        </div>
+    </article>
```

Comment:
- [ ] [OPTIONAL] This is labeled optional because the previous reviewer missed it. You already have an array of all the guest speakers, therefore the easier way to do this would be to loop over the array and render the speaker articles dynamically instead of hard-coding it this way 👍